src/primitives/transaction.h

```diff
@@ -24,6 +24,7 @@ enum OutputTypes
     OUTPUT_CT               = 2,
     OUTPUT_RINGCT           = 3,
     OUTPUT_DATA             = 4,
+    OUTPUT_ZC               = 5,
```

Comment:
A zerocoin output is a standard output as of now. I think we should leave this part of the change out, unless we do a full analysis of what problems adding an additional output type could cause in the validation code.

Reply:
Removed in 418743623784a5ae2a366f0a9a65af1cfc921d20